tests/data-files/landsat/readme.md

```diff
@@ -0,0 +1,9 @@
+# Notes on data in this folder
+
+Files sourced from the landsatlook sat-api implementation, accessible here:
+`https://landsatlook.usgs.gov/sat-api/`
+
+Downloaded metadata from this scene:
+`https://landsatlook.usgs.gov/sat-api/collections/landsat-c2l2-sr/items/LC08_L2SR_081119_20200101_20200823_02_T2`
+
+Run `make download` to get these files. Run `make clean` to delete them.
```

Comment:
Seems like these files are checked into the repo with this PR, is the makefile needed? If it's not, probably best to remove - I think the references to the download locations is enough

Reply:
It's not needed, but is pretty convenient!

You sure you want it gone?